src/transaction.ts

```diff
@@ -645,7 +747,87 @@ class Transaction extends DatastoreRequest {
         }
       );
     };
-    return new Promise(promiseFunction);
```

Comment:
```
  (err, resp) => {
    resolve({
      err,
      resp,
    });
  }
```

Is the function expression really needed here? I though resolve was already a function. Couldn't this block just be replaced by `resolve`?